templates/Dockerfile.template

```diff
@@ -1,4 +1,29 @@
 FROM {{ .From }}
```

Comment:
Please consider using templates feature as in `https://github.com/GoogleCloudPlatform/rabbitmq-docker/blob/master/versions.yaml`